packages/material-ui/src/Collapse/Collapse.js

```diff
@@ -27,7 +27,6 @@ export const styles = (theme) => ({
     height: 'auto',
     overflow: 'visible',
     '&$horizontal': {
-      height: 'initial',
```

Comment:
https://stackoverflow.com/questions/20823105/use-initial-width-for-element-not-working-in-ie